src/build/prefetch.ts

```diff
@@ -42,8 +42,9 @@ export class PrefetchTransform extends Transform {
     type: 'import' | 'prefetch'
   ) {
     let contents = file.contents.toString();
-    let ast = parse5.parse(contents);
-    let head = dom5.query(ast, dom5.predicates.hasTagName('head'));
+    const ast = parse5.parse(contents);
+    // parse5 always produces a <head> element
+    const head = dom5.query(ast, dom5.predicates.hasTagName('head'))!;
```

Comment:
+1 to `const`ifying. Looks like typescript finds more and more places where they're able to do better type inference with `const` over `let`. (yes, `const`-like behavior can ~always be inferred but the convenience of not having to do that does seem to be paying out for the tsc team) https://github.com/Microsoft/TypeScript/pull/10676